mirascope/chat/tools.py

```diff
@@ -67,6 +120,9 @@ def tool_schema(cls) -> ChatCompletionToolParam:
     def from_tool_call(cls, tool_call: ChatCompletionMessageToolCall) -> OpenAITool:
         """Extracts an instance of the tool constructed from a tool call response.
 
+        Takes `ChatCompletionMessageToolCall` from an OpenAI chat completion response,
```

Comment:
This doesn't add anything compared to above line.

I think it would be better to instead describe what it actually does, as in takes the arguments from the function of the given tool call and creates the openaitool instance etc.